cmd/tk/toolCharts.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/go-clix/cli"
+	"github.com/grafana/tanka/pkg/helm"
+	"gopkg.in/yaml.v2"
+)
+
+func chartsCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "charts",
+		Short: "Declarative vendoring of Helm Charts",
+	}
+
+	cmd.AddCommand(
+		chartsInitCmd(),
+		chartsAddCmd(),
+		chartsVendorCmd(),
+		chartsConfigCmd(),
+	)
+
+	return cmd
+}
+
+func chartsVendorCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "vendor",
+		Short: "Download Charts to a local folder",
+	}
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		c, err := loadChartfile()
+		if err != nil {
+			return err
+		}
+
+		return c.Vendor()
+	}
+
+	return cmd
+}
+
+func chartsAddCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "add [chart@version] [...]",
+		Short: "Adds Charts to the chartfile",
+	}
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		c, err := loadChartfile()
+		if err != nil {
+			return err
+		}
+
+		return c.Add(args)
+	}
+
+	return cmd
+}
+
+func chartsConfigCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "config",
+		Short: "Displays the current manifest",
+	}
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		c, err := loadChartfile()
+		if err != nil {
+			return err
+		}
+
+		data, err := yaml.Marshal(c.Manifest)
+		if err != nil {
+			return err
+		}
+
+		fmt.Print(string(data))
```

Comment:
could we colourise this?

Reply:
Not easily. The only yaml highlighter library I am aware of (alecthomas/chroma) doesn't handle yaml well, especially not on light schemed terminals.

We actually removed it from the project for that exact reason some time ago (were coloring diff using it)

We could look into writing a custom yaml highlighter some day